modules/ROOT/pages/tutorial/tutorial-sso-configuration.adoc

```diff
@@ -234,33 +234,33 @@ For details, see https://learn.microsoft.com/en-us/azure/active-directory/develo
 dbms.security.oidc.azure.claims.username=sub
 ----
 
-==== Map Azure groups to Neo4j roles
+==== Map Entra groups to Neo4j roles
 
-Decide whether you want to use Azure AD Groups directly or Azure App Roles.
+Decide whether you want to use Entra AD Groups directly or Entra App Roles.
 
 Using AD Groups directly might be convenient if you already have users assigned to relevant AD Groups and want to perform Group-to-Role mapping in Neo4j settings.
 
-Azure App Roles allow a layer of separation between Neo4j roles and AD Groups.
+Entra App Roles allow a layer of separation between Neo4j roles and AD Groups.
 When App Roles are used, only the roles relevant to Neo4j are sent in the JWT token.
 This prevents leaking permissions between applications.
 JWT tokens also have a limitation of 200 roles per token per user, which can be avoided by sending only the relevant App Roles.
 
-Details about Azure App Roles can be found in the https://learn.microsoft.com/en-us/azure/active-directory/develop/howto-add-app-roles-in-azure-ad-apps[Microsoft documentation].
+Details about Entra ID App Roles can be found in the https://learn.microsoft.com/en-us/entra/identity-platform/howto-add-app-roles-in-apps[Microsoft documentation].
 
-==== Using Azure AD Groups directly
+==== Using Entra AD Groups directly
```

Comment:
I am not sure if they still call them AD groups.

Reply:
You're right! It should be just Entra. I will fix that. Thanks!